extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -127,6 +129,24 @@ public function addTitle( $id, $title ) {
 		$this->titles[$id] = $title;
 	}
 
+	public function addVote( $id, $userId ) {
+		if ( !empty( $this->votes[$id] ) ) {
+			$voters = $this->votes[$id];
+
+			//There are weird cases when single user did multiple votes on the same post.
+			if (in_array($userId, $voters)) {
+				return false;
+			}
+
+			$voters[] = $userId;
+			$this->votes[$id] = $voters;
+			return true;
+		}
+
+		$this->votes[$id] = [$userId];
```

Comment:
why the userId is added to the array as an array instead of single value?

Reply:
Because single post represented by $id could have few voters, that's why I need array of voters to determine if there are duplicates.

Reply:
oh right, nvm then